sled-agent/src/illumos/zone.rs

```diff
@@ -5,31 +5,71 @@
 //! API for interacting with Zones running Propolis.
 
 use ipnetwork::IpNetwork;
-use omicron_common::api::external::Error;
 use slog::Logger;
 use std::net::SocketAddr;
 use uuid::Uuid;
 
+use crate::illumos::dladm::VNIC_PREFIX_CONTROL;
 use crate::illumos::zfs::ZONE_ZFS_DATASET_MOUNTPOINT;
 use crate::illumos::{execute, PFEXEC};
 
-const BASE_ZONE: &str = "propolis_base";
+const PROPOLIS_BASE_ZONE: &str = "oxz_propolis_base";
+const STORAGE_BASE_ZONE: &str = "oxz_storage_base";
 const PROPOLIS_SVC_DIRECTORY: &str = "/opt/oxide/propolis-server";
+pub const CRUCIBLE_SVC_DIRECTORY: &str = "/opt/oxide/crucible-agent";
+pub const COCKROACH_SVC_DIRECTORY: &str = "/opt/oxide/cockroachdb";
 
+const DLADM: &str = "/usr/sbin/dladm";
 const IPADM: &str = "/usr/sbin/ipadm";
 const SVCADM: &str = "/usr/sbin/svcadm";
 const SVCCFG: &str = "/usr/sbin/svccfg";
 const ZLOGIN: &str = "/usr/sbin/zlogin";
 
-pub const ZONE_PREFIX: &str = "propolis_instance_";
+// TODO: These could become enums
```

Comment:
Too bad there is not some easy way to have a const build a &str from another const, so you could have something like this pseudocode:
```
ZONE_PREFIX = "oxz_"
PROPOLIS_ZONE_PREFIX = ZONE_PREFIX + "propolis_instance_" 
```